x-pack/test/security_solution_cypress/es_archives/unmapped_fields/mappings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "index",
+  "value": {
+    "aliases": {
+      "mydata-7.12.0": {
+        "is_write_index": true
+      }
+    },
+    "index": "unmapped-7.12.0-2021.03.10-000001",
+    "mappings": {
+      "dynamic": false,
+      "properties":{
+        "@timestamp":{
+          "type":"date"
+        },
+        "mydestination":{
+          "properties":{
+            "ip":{
+              "index": "false",
+              "type":"keyword"
+            }
+          }
+        }
+      }
+    }
+    }
```

Comment:
I would indent by using format on save within your IDE as a good time saver and way to keep these brackets nice and clean.